### PR TITLE
[docs] git-workflow.md corrections

### PR DESCRIPTION
**Pull request by vchenin**
phpunit has option '--list-groups', but not '--groups'
issue number used for examples is 999, but in commit message is 42
replace names git and github with Git and GitHub
[skip ci]

### CODE REVIEW

File: docs/internals-ja/git-workflow.md [hunk 1]
```diff
@@ -55,15 +55,17 @@ phpunit をグローバルにインストールしていない場合は、代り
...
 ### エクステンション
 
 エクステンションに取り組むためには、エクステンションのレポジトリをクローンする必要があります。
 私たちは、あなたに代ってそれをするコマンドを作っています。
 
-    php build/build dev/ext <extension-name>
-    
+```
+php build/build dev/ext <extension-name>
+```
```
Comment (cebe) [inline docs/internals-ja/git-workflow.md]: why? 4 space indentation is totally valid for code and more readable for one liners imo.

Comment (samdark) [inline docs/internals-ja/git-workflow.md]: Doesn't matter. The rest of changes are no doubt valid so I'm merging it.

**Comment by samdark** [comment]
Thanks!

File: docs/internals-ja/git-workflow.md (hunk 1 above)
Comment (vchenin) [inline docs/internals-ja/git-workflow.md]: @cebe I forgot this
but in most documentation which i had translate was used triple backtick, even for one liners
so it's better for uniformity

**Pull request merged**